platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java

```diff
@@ -433,31 +242,24 @@ private void createSecurityToken(SoapVersion version, SOAPFactory soapFactory,
             if (principals != null) {
                 SecurityAssertion securityAssertion = principals.oneByType(SecurityAssertion.class);
                 if (securityAssertion != null) {
-                    SecurityToken securityToken = securityAssertion.getSecurityToken();
-                    Element samlElement = securityToken.getToken();
-                    SOAPElement samlAssertion = null;
-                    try {
-                        samlAssertion = soapFactory.createElement(samlElement);
-                        securityHeader.addChildElement(samlAssertion);
-
-                    } catch (SOAPException e) {
-                        LOGGER.error("Unable to convert SecurityToken to SOAPElement.", e);
-                    }
+                    securityToken = securityAssertion.getSecurityToken();
                 } else {
-                    LOGGER.warn("Subject did not contain a security assertion, could not assertion"
-                            + "to security header.");
+                    LOGGER.warn(
+                            "Subject did not contain a security assertion, could not assertion to security header.");
```

Comment:
It looks like this message is missing a word.
"Subject did not contain a security assertion, could not add assertion to SoapMessage."